docs/src/main/paradox/directives/linking.md

```diff
@@ -104,8 +104,20 @@ the apidocs, and linking to the non frames version. This can be controlled using
 property, setting it to `frames` for frames style linking, where rendered links link to the frames
 version of the javadocs, passing the class in the query parameter, and `direct` for linking direct
 to the classes page. The link style defaults to `direct` if the `java.specification.version` system
-property indicates Java 9+, since the JDK 9 Javadoc tool does not generate framed api docs. Otherwise,
-for Java 1.8 and earlier, it defaults to `frames`.
+property indicates Java 11+, since by in JDK 11, the Javadoc tool does not generate framed api docs. 
+Otherwise, for Java 10 and earlier, it defaults to `frames`.
```

Comment:
Linking to frames from 11+ instead of 9+ seems like an independent change - OK if you're confident it's the right thing though :D